cpu/cpustress.py

```diff
@@ -83,12 +83,10 @@ def __clear_dmesg():
     @staticmethod
     def __error_check():
         ERROR = None
-        logs = (process.system_output(
-            "dmesg -T --level=alert,crit,err,warn", ignore_status=True, shell=True, sudo=True)).split()
-        for log in logs:
-            for error in errorlog:
-                if error in log:
-                    ERROR = 'Test resulted %s in syslogs"' % error
+        logs = process.system_output("dmesg -T --level=alert,crit,err,warn")
```

Comment:
Suggestion: `dmesg -Txl 1,2,3,4`

Reply:
I don't see any value add, Please let me know if any. 

However both the command gives same output, Hence I have modified it. Still I am interested to know how different is this from the command with actual names.

Reply:
@harihare There is no difference, except for the addition of `x`, which specifies the level.